app/views/admin/councillor_contributions/_details.html.arb

```diff
@@ -0,0 +1,13 @@
+table do
+  tr do
+    th "Contributor Name"
+    th "Contributor Email"
+    th "Authority"
+  end
+  tr do
+    td a contributor.name, href:admin_contributor_path
+    td contributor.email
+    "#{contributor.email}"
+    td a authority.full_name
```

Comment:
This doesn't link to anything when I test locally. What should it be going to and why?